cluster-provision/okd/scripts/run.sh

```diff
@@ -50,19 +57,27 @@ for domain in $(virsh list --name --all); do
 	[[ $domain =~ worker ]] && virt-xml --edit --memory ${WORKERS_MEMORY} $domain && virt-xml --edit --vcpu ${WORKERS_CPU} $domain
 
 	virsh start $domain
-
 done
 
 while [[ "$(virsh list --name --all)" != "$(virsh list --name)" ]]; do
     sleep 1
 done
 
 export KUBECONFIG=/root/install/auth/kubeconfig
-oc config set-cluster test-1 --server=https://127.0.0.1:6443
-oc config set-cluster test-1 --insecure-skip-tls-verify=true
+
+# update bashrc to make life easier
```

Comment:
As a user running this locally, I would not want this script to change my bashrc.

Reply:
its bashrc in the container only
it will allow you to connect to the container and run oc commands there and shortcuts
without exporting the KUBECONFIG and without writing long commands again and again
all this even before cluster up is finished
for example we want a lot of time to see what pods arent running

is it ok ?
Daniel for example like the idea
repetitive commands are sweat and error prune imo